.github/workflows/plugin_install.yml

```diff
@@ -0,0 +1,71 @@
+name: Plugin Install
+
+on: [push, pull_request, workflow_dispatch]
+
+env:
+  OPENSEARCH_VERSION: "1.3.0"
+
+jobs:
+  plugin_install:
+    name: Plugin Install
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        jdk: [8, 11, 14]
+
+    steps:
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.jdk }}
+
+    - name: Checkout security
+      uses: actions/checkout@v2
+
+    - name: Build
+      run: ./gradlew clean assemble -Dbuild.snapshot=false
+
+    - name: Download OpenSearch Core
+      run: |
+        wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/${{env.OPENSEARCH_VERSION}}/latest/linux/x64/builds/opensearch/dist/opensearch-min-${{env.OPENSEARCH_VERSION}}-linux-x64.tar.gz
+        tar -xzf opensearch-*.tar.gz
+        rm -f opensearch-*.tar.gz
+
+    - name: Move and rename security plugin for installation
+      run: mv build/distributions/opensearch-security-${{env.OPENSEARCH_VERSION}}.0.zip opensearch-security.zip
+
+    - name: Run OpenSearch with plugin
+      run: |
+        cat > os-ep.sh <<EOF
```

Comment:
Is it necessary to build Docker image and run OpenSearch cluster with Docker? With security plugin installed and demo configuration configured, can't OpenSearch cluster start without Docker.

Reply:
I followed the model we use for integration tests for the security-dashboard-plugin, if you have thoughts on how we can shift this functionality to another model I think that would be great follow up work.